demos/python_demos/seq2seq_translation/README.md

```diff
@@ -24,6 +26,53 @@ cd demos/python_demos/seq2seq_translation
 docker run -it --rm -p 9000:9000 -v ${PWD}/servable:/workspace openvino/model_server:py --config_path /workspace/config.json --port 9000
 ```
 
+## Running Model Server on Windows (experimental)
+
+### Build Model Server binary with Python enabled
+For the Model Server to include embedded Python interpreter you need to [build the binary from scratch](../../../docs/windows_developer_guide.md). Note the `--define MEDIAPIPE_DISABLE=0 --define PYTHON_DISABLE=0` part in `bazel build` command that specifies that both MediaPipe and Python support will be included (Python support implementation is based on MediaPipe). 
```

Comment:
Note the `--define MEDIAPIPE_DISABLE=0 --define PYTHON_DISABLE=0` part in `.bazelrc` file in # Windows config default flags section - build:windows,  that specifies that both MediaPipe and Python support will be included (Python support implementation is based on MediaPipe).

Reply:
Removed entirely